env/requirements_tensorflow-gnn.yml

```diff
@@ -0,0 +1,16 @@
+name: data-science-cop-tfgnn
+channels:
+  - conda-forge
+dependencies:
+  - python=3.10
+  - jupyterlab=4.2.3
+  - ipykernel=6.29.5
+  - pandas=2.2
+  - iris=3.8
+  - scikit-learn=1.4
+  - networkx=3.3
+  - xarray=2024.6.0
+  - pip=24.0
+  - pip: 
+    - tensorflow==2.12
```

Comment:
Tensorflow-gnn isn't available with conda (at least right now) so I thought it might be reasonable to into install both tensorflow and tensorflow-gnn with the same package manager to try and prevent issues. I'm pretty sure that I tried using conda for tensorflow and didn't have any problems so I don't believe it will make a major difference.